front/src/Stores/ChatStore.ts

```diff
@@ -33,6 +43,42 @@ function getAuthor(authorId: number): PlayerInterface {
     return author;
 }
 
+function createWritingStatusMessageStore() {
+    const { subscribe, update } = writable<WritingMessage[]>([]);
```

Comment:
I think the only thing you need is an array of players.

```suggestion
    const { subscribe, update } = writable<PlayerInterface[]>([]);
```